package.json

```diff
@@ -109,6 +109,7 @@
     "@types/chai": "4.3.5",
     "@types/jest": "29.5.3",
     "@types/node": "18.17.0",
+    "@typescript-eslint/eslint-plugin": "^6.2.0",
     "chai": "4.3.7",
```

Comment:
Curious why this is needed as I thought we were removing direct ESLint dependencies in favour of the `eslint-config` package? It's a dependency there: https://github.com/Pocket/eslint-config/blob/main/package.json

Reply:
ah, this probably sneaked in when I was running lint locally and needed to install the package. will remove it!